VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -102,13 +102,13 @@ public void notifyNetworkConnect(){
 
     @Override
     protected void onCreate(Bundle arg0) {
-        super.onCreate(arg0);
-
         // for landscape player use full screen theme, otherwise, only hide title bar
         if (isLandscape()) {
             setTheme(android.R.style.Theme_NoTitleBar_Fullscreen);
         }
 
+        super.onCreate(arg0);
```

Comment:
In my opinion we don't need to change the theme here at all. As mentioned in the commit message, `CourseUnitNavigationActivity` was the only subclass of `BaseFragmentActivity` to support landscape orientation (as well as probably the intended `Activity` for this theme), and it handled hiding the action bar itself properly (as evidenced by the fact that the `RoboFragmentActivity` inheritance didn't cause any actual issue other than to cause the test to fail. I just patched the code so that the tests would work.